app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide :title, 'Help' %>
+<h1>Help</h1>
+<p>
+	Get help on the Ruby on Rails Tutorial at the
```

Comment:
Tìm hiểu `I18n in rails ` nè

Self check lại indent và I18n, link_to cho toàn bộ các file html bên dưới